### ISSUE TITLE
Accessibility issues round-up

### ISSUE THREAD

**Issue by dan-bowerman** [ramp4-pcar4/story-ramp#260]
`1.	The header below is currently markup as <h2>, but it should markup as <h3>`

> a.	Kespukwitk / Southwest Nova Scotia
> b.	Kespukwitk / Southwest Nova Scotia: Action on the Ground
> c.	2.1 Wolastoq / Saint John River
> d.	3.1 PEI Forested Landscape
> e.	3.2 PEI Forested Landscape: Action on the Ground
> f.	4.1 St. Lawrence Lowlands
> g.	4.2 St. Lawrence Lowlands: Action on the Ground
> h.	5.1 Long Point Walsingham Forest
> i.	5.2 Long Point Walsingham Forest: Action on the Ground
> j.	6.1 Mixed Grass Prairie
> k.	6.2 Mixed Grass Prairie: Action on the Ground
> l.	7.1 South of the Divide
> m.	7.2 South of the Divide: Action on the Ground
> n.	8.1 Summit to Sage
> o.	8.2 Summit to Sage: Action on the Ground
> p.	9.1 Dry Interior British Columbia
> q.	9.2 Dry Interior British Columbia: Action on the Ground
> r.	10.1 Southwest British Columbia
> s.	10.2 Southwest British Columbia: Action on the Ground
> t.	11.1 Yukon South Beringia
> u.	11.2 Yukon South Beringia: Action on the Ground
> v.	Specific Priority Places
> w.	Background Reading

DanB: Can we make these headers h3 instead of h2? I'm not sure how this affects anyone with impairments, but let's fix it in the platform. 

`2.	Title case header vs Sentence case header `

> a.	All headers should be in Sentence case, however there are some exception case apply, here is the exception case link: 
> i.	https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/canada-content-style-guide.html#wp4-1 
> ii.	For example: The last header “Want to Learn More?” should change to “Want to learn more?” 

DanB: I will address this. 

`3.	Static textual content`

> a.	All the Static Text are compliant to the Contrast ratio. They are all Black Text on White Background

DanB: Doesn't seem like there is anything to do here.

`4.	Photographs`

> a.	The contrast ratio requirement for accessibility is only cover text or image of text, this criteria is not cover the photographs, so all the photographs are good in the story map.

DanB: Doesn't seem like there is anything to do here.

`5.	Italic text`

> a.	We should follow the Canada.ca content style guide, Italic text is only allowed for a Citation, or to identify different language.
> b.	For example, in the webpage, all photo credit are marked up as italic text like this “Photo Credit: Darcy Henderson”

DanB: I will remove the italics. This is the way the existing StoryMap had laid out the text.
